sdk/core/azure-core-amqp/pom.xml

```diff
@@ -14,7 +14,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-core-amqp</artifactId>
-  <version>1.5.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
+  <version>1.5.0</version> <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
```

Comment:
Missing update on the [core-amqp-readme](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/core/azure-core-amqp#getting-started)